### PR TITLE
hardening/672_document_maximum_batch_size

### PR DESCRIPTION
**Pull request by pcoello25**
* Fixes issue #672 
* No test are required
* Assignee @frbattid

### CODE REVIEW

**Comment by frbattid** [comment]
CHANGES_NEXT_RELEASE update is required.

File: doc/flume_extensions_catalogue/orion_dynamodb_sink.md [hunk 1]
```diff
@@ -130,7 +130,7 @@ If `table_type=table-by-service-path` and `attr_persistence=colum` then `OrionDy
 | enable\_grouping | no | false | <i>true</i> or <i>false</i> |
 | table_type | no | table-by-destination | <i>table-by-destination</i> or <i>table-by-service-path</i> |
 | attr\_persistence | no | row | <i>row</i> or <i>column</i>
-| batch\_size | no | 1 | Number of events accumulated before persistence |
+| batch\_size | no | 1 | Number of events accumulated before persistence (Max 25, check [Amazon Web Services Documentation](http://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Limits.html) for more information) |
```
Comment (frbattid) [inline doc/flume_extensions_catalogue/orion_dynamodb_sink.md]: max --> maximum... this is a formal document :)

**Comment by pcoello25** [comment]
Fixed in #674

File: CHANGES_NEXT_RELEASE [hunk 2]
```diff
@@ -1,3 +1,4 @@
 - [FEATURE] Add batching support to OrionTestSink (#572)
 - [HARDENING] Add the minimum HW requirements (#640)
 - [HARDENING] Rename the HTTP and Flume header constants (#662)
+- [HARDENIND] Add the maximum accepted batch size in DynamoSink (#672)
```
Comment (frbattid) [inline CHANGES_NEXT_RELEASE]: HARDENIND --> HARDENING

**Comment by frbattid** [comment]
LGTM

**Pull request merged**